projects/core/src/modules/numeric/numeric.pipe.ts

```diff
@@ -58,6 +69,16 @@ export class SkyNumericPipe implements PipeTransform, OnDestroy {
   }
 
   public transform(value: number, config?: NumericOptions): string {
+
+    /* If the value and locale are the same as the last transform then return the previous value
+    instead of reformatting. */
+    if (this.formattedValue &&
+      value === this.rawValue &&
+      this.areConfigsEqual(this.lastConfig, config) &&
+      (config?.locale || this.providerLocale === this.lastTransformLocale)) {
```

Comment:
I'd take a look at [what Paul did](https://github.com/blackbaud/skyux-i18n/blob/master/projects/i18n/src/modules/i18n/resources.pipe.ts#L43) for `skyAppResources` and `skyLibResources` pipes. If you generated a cache key using `JSON.stringify` you could avoid adding complex logic to determine if the config has changed. To ensure that the stringification always checks the config's keys in alphabetical order, you could do something like:

```
JSON.stringify(config, Object.keys(config).sort())
```

Also, by [adding the async `getLocaleInfo()` call to the `transform()` method](https://github.com/blackbaud/skyux-i18n/blob/master/projects/i18n/src/modules/i18n/resources.pipe.ts#L46), you would only be accessing the resources service when the pipe is used, instead of when the pipe's class is constructed on the app's initialization.

Reply:
Fixed the cache mechanism and decided to leave the `getLocalInfo` in the constructor due to how intertwined it is with the numeric service.

Reply:
Doh. Done.